scale/job/execution/running/tasks/job_task.py

```diff
@@ -1,12 +1,12 @@
 """Defines the class for a job execution job task"""
 from __future__ import unicode_literals
 
-from job.execution.running.tasks.base_task import Task
+from job.execution.running.tasks.exe_task import JobExecutionTask
 from job.resources import NodeResources
 
 
-class JobTask(Task):
-    """Represents a job execution job task (runs the actual job/algorithm)
+class JobTask(JobExecutionTask):
```

Comment:
I agree it's confusing. The job execution is split into three tasks: pre-task, job-task, and post-task. The job-task is confusingly named, but I didn't have a better alternative at the time. Maybe main-task? I am definitely open to better suggestions.